conf/references/mlss_conf.xml

```diff
@@ -11,39 +11,44 @@
       <!-- Metazoa -->
       <genome name="drosophila_melanogaster"/>
       <genome name="caenorhabditis_elegans"/>
-      <genome name="octopus_bimaculoides"/>
       <!-- Fungi -->
       <genome name="saccharomyces_cerevisiae"/>
       <!-- Plants -->
       <genome name="arabidopsis_thaliana"/>
-      <genome name="triticum_aestivum"/>
+      <genome name="oryza_sativa"/>
       <!-- Protists -->
       <genome name="plasmodium_falciparum"/>
     </collection>
 
+    <collection name ="shared_vertebrates">
+      <base_collection name="shared"/>
+      <genome name="mus_musculus"/>
+      <genome name="xenopus_tropicalis"/>
```

Comment:
```suggestion
      <genome name="xenopus_tropicalis"/>
      <genome name="ciona_intestinalis"/>
```

Reply:
ciona_intestinalis cannot be in the shared_vertebrates as it is not in the mammalia. The reason why it is not in the mammalia is because there is too many important species in the mammalia belonging to the a same subclade and so I preferred to remove a few outgroups rather than representative of some mammalian subclades.

Reply:
So  is 40 a maximum cap now and not an aim/goal? I think the use and importance of outgroups in reference species needs to be looked into further. Outgroups are essential to comparative analyses in evolutionary biology. In my experience, outgroups should equal or outweigh the clade of interest. Where that is not possible, the disparity in outgroups should be considered, for example, looking at different levels of outgroup relationship. In this case, we could consider the closely related outgroups to mammalia being non-mammalian vertebrates, and then we should hop further and look to non-vertebrate chordates. Ideally, every clade in vertebrates should be compared to at least one outgroup in each of the deuterostome phyla/subphyla - this might not be possible given a cap, but I leave this here.

Reply:
Yes the cap is important as it makes sure that we are able to process the query genome on a given time. I agree with you about the importance of outgroups, and I  see this will need more discussion to formerly define how we select outgroups, at the moment I was more focusing on the representative of the clade targeted, but yes we need to define a formal way to identify outgroups . In the example of vertebrates,  having a deuterostome for all vertebrates collections may no be always needed especially for collection that are deep into the clade (if we consider to make a primate collection or rodent one for instance).